Content.Client/Power/PowerMonitoringWindow.xaml.Widgets.cs

```diff
@@ -478,6 +505,31 @@ public PowerMonitoringButton()
 
         MainContainer.AddChild(NameLocalized);
 
+        BatteryLevel = new ProgressBar()
+        {
+            HorizontalAlignment = HAlignment.Right,
+            SetWidth = 60f,
+            Margin = new Thickness(15, 7, 0, 7),
+            MinValue = 0,
+            MaxValue = 100,
+            Page = 0,
+            Value = 0,
+            Visible = false,
+        };
+
+        MainContainer.AddChild(BatteryLevel);
+
+        BatteryPercentage = new Label()
+        {
+            HorizontalAlignment = HAlignment.Right,
+            SetWidth = 45f,
```

Comment:
You should probably set a MinWidth instead so the layout doesn't break when we get the ability for users to select custom fonts.